WalletWasabi.Gui/Behaviors/PasteAddressOnClickBehavior.cs

```diff
@@ -96,25 +125,47 @@ protected override void OnAttached()
 				AssociatedObject.GetObservable(InputElement.PointerReleasedEvent).Subscribe(async pointer =>
 				{
 					var uiConfig = Application.Current.Resources[Global.UiConfigResourceKey] as UiConfig;
-					if (!(uiConfig.Autocopy is true))
+					if (uiConfig.Autocopy is false)
 					{
 						return;
 					}
 
 					switch (MyTextBoxState)
 					{
 						case TextBoxState.AddressInsert:
-							var result = await IsThereABitcoinAddressOnTheClipboardAsync();
+							var result = await ProcessClipboardAsync();
 
 							if (result.isAddress)
 							{
-								AssociatedObject.Text = result.address;
+								AssociatedObject.Text = result.url.Address.ToString();
 							}
 							MyTextBoxState = TextBoxState.NormalTextBoxOperation;
```

Comment:
That is where to Command should called if there is any. Should be called CommandOnAddress or something like this and pass the required parameters through the arguments.